### PR TITLE
fix: "Enter location" recyclerview bottom margin

### LINKED ISSUE #2470: "Enter Location" recyclerview margin from bottom
**Describe the bug**
The recyclerview in "Enter location" is not having proper margins from the bottom which make it look not so good.
**To Reproduce**

**Screenshots**

WhatsApp Image 2019-12-10 at 12 01 41 AM

**Smartphone Info:**

|               | |
|---------------|-|
|Device         |[Redmi Note 5]|
|Android Version            |[Oreo 8.1 |

**Additional context**

**Would you like to work on the issue?**

- [✓] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by VarunSaini23**
Fixes #2470 
Changes: 
Added android:layout_marginBottom="@dimen/layout_margin_medium"
attribute in the recyclerview.

Screenshots for the change:
WhatsApp Image 2019-12-10 at 11 30 06 AM

### CODE REVIEW

File: app/src/main/res/layout/fragment_search_location.xml [hunk 1]
```diff
@@ -91,6 +91,7 @@
                 android:layout_marginTop="@dimen/layout_margin_medium"
                 android:layout_marginEnd="@dimen/layout_margin_medium"
                 android:layout_marginRight="@dimen/layout_margin_medium"
+                android:layout_marginBottom="@dimen/layout_margin_medium"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_search_location.xml]: Change all 4 to just margin

Comment (VarunSaini23) [reply app/src/main/res/layout/fragment_search_location.xml]: Okay

Comment (VarunSaini23) [reply app/src/main/res/layout/fragment_search_location.xml]: > Change all 4 to just margin

Done

File: app/src/main/res/layout/fragment_search_location.xml [hunk 2]
```diff
@@ -88,9 +88,7 @@
                 android:id="@+id/placeSuggestionsCard"
                 android:layout_marginStart="@dimen/layout_margin_medium"
                 android:layout_marginLeft="@dimen/layout_margin_medium"
```
Comment (nikit19) [inline app/src/main/res/layout/fragment_search_location.xml]: Remove these 2 also

Comment (VarunSaini23) [reply app/src/main/res/layout/fragment_search_location.xml]: @nikit19 Done

**Pull request merged**